integration-tests/main/src/test/java/io/quarkus/example/test/JaxRSTestCase.java

```diff
@@ -135,4 +135,17 @@ public void testFromJson() {
                 .body("name", is("my entity name"),
                         "value", is("my entity value"));
     }
+
+    @Test
+    public void testOpenApiSchemaResponse() {
+        RestAssured.when().get("/test/openapi/responses").then()
+                .body("name", is("my openapi entity name"));
+    }
```

Comment:
You're not testing the `"/openapi/responses/{version}"` path.

Otherwise, it looks good.

Could you rebase and take care of it?

Reply:
Great. I have rebased the master and force pushed to the branch.